lib/suse/connect/templates/extension.text.erb

```diff
@@ -0,0 +1,6 @@
+<%= render 'extension_item.text', { extension: extension, any_installed: any_installed, level: level} %>
+<% if extension[:extensions] -%>
+<%   extension[:extensions].each do |sub_extension| -%>
```

Comment:
You can remove the first `if extension[:extensions]` by changing the loop to: `extension.fetch(:extensions, []).each do |sub_extension|`

Reply:
Ah nice yes I do this